app/lib/swagger_docs.rb

```diff
@@ -54,6 +54,11 @@ class SwaggerDocs
     "<p>Note: for submissions after MTR Phase 2 is implemented, “disregarded payments” will be extended to cover “Modern Slavery Victim Care Contract (MSVCC) financial support payments”, and no longer cover “Back to Work Bonus payments”</p>",
   ].join
 
+  INACCESSIBLE_CAPITAL_TEXT = [
```

Comment:
@starswan We need to add this text to **properties** and **vehicles** as per the comments